index.coffee

```diff
@@ -32,7 +33,11 @@ module.exports = (grunt) ->
 
     {certificateFile, certificatePassword} = config
 
-    appMetadata = grunt.file.readJSON(path.join(appDirectory, 'resources', 'app', 'package.json'))
+    appDir = path.join(appDirectory, 'resources', 'app')
```

Comment:
Done; however, I personally think the appDirectory itself could use some renaming to avoid the opposite confusion.  But that's a different (larger) conversation, since it would break the interface.